File: java/0009-palindrome-number/Main.java
public class Main {
    public static void main(String[] args) {
        Solution solution = new Solution();

        System.out.println("true: " + solution.isPalindrome(121));
        System.out.println("false: " + solution.isPalindrome(-121));
        System.out.println("false: " + solution.isPalindrome(10));
    }
}

File: c++/0009-palindrome-number/main.cpp
#include <cstdlib>
#include <iostream>

#include "solution.cpp"

using namespace std;

int main() {
    Solution solution;

    string solution1 = solution.isPalindrome(121) == true ? "true" : "false";
    string solution2 = solution.isPalindrome(-121) == true ? "true" : "false";
    string solution3 = solution.isPalindrome(19) == true ? "true" : "false";

    cout << "true: " << solution1 << endl;
    cout << "false: " << solution2 << endl;
    cout << "false: " << solution3 << endl;

    return EXIT_SUCCESS;
}
